docs/content/preview/releases/ybdb-releases/v2024.2.md

```diff
@@ -0,0 +1,476 @@
+
+* [Semi-automatic xCluster replication](/stable/deploy/multi-dc/async-replication/async-replication-transactional/). Provides simplified management of YSQL transactional xCluster replication. xCluster management operations are now only required when adding and removing databases from replication. DDL changes can now be made by any database administrator or user with database permissions, and no longer require SSH access or intervention by an IT administrator.
+
+* [xCluster observability](/stable/yugabyte-platform/alerts-monitoring/alert-policy-templates/#xcluster-config-tables-are-in-bad-state). Improved ability for users to monitor xCluster metrics like Replication Lag, Safetime Lag, and replication errors. {{<tags/feature/ea>}}
+
+* [Follower reads](/stable/explore/going-beyond-sql/follower-reads-ysql/). Added support for setting `yb_read_from_followers=true` inside a transaction block using SET LOCAL. This allows greater convenience when setting follower reads for single transactions. You must set `yb_read_from_followers` before any statement is executed in the block.
+
+* [Improved query performance](/stable/reference/configuration/yb-tserver/#pg-client-use-shared-memory). Implemented shared memory for PostgreSQL to TServer communication to minimize latency and enhance query performance by reducing communication overhead. This enhancement is part of [Enhanced PostgreSQL compatibility mode](../../../explore/ysql-language-features/postgresql-compatibility/#enhanced-postgresql-compatibility-mode).
+
+<!-- IDEA-1093 is moved to 2024.2.1
+* [Skip column updates](/stable/reference/configuration/yb-tserver/#yb-skip-redundant-update-ops). Avoids updating columns which are included in any index and foreign key constraints when the columns values are not changed. -->
+
+* SELECT query performance. Enables the batching for locking the rows and thus helps improve the performance of the queries which lock more than 1 row using SELECT FOR UPDATE.
+<!-- No docs needed-->
+
+* Sequential scan speed. Improved ANALYZE and sequential scan performance by leveraging OS-level read-ahead, resulting in up to 8x faster ANALYZE operations.
+<!-- IDEA-1878. Unsure about docs -->
+
+* ALTER support. Added support for miscellaneous ALTER TABLE DDLs.
+<!-- DOC-528 pending -->
+
+* Catalog caching. Added ability to find catalog tables that can be pre-loaded to improve latency for the first query in a connection, or after a DDL change in the cluster. {{<tags/feature/ea>}}
+<!-- DOC-526, won't make it to 2024.2 -->
+
+* CDC connector on Confluent Hub. YugabyteDB CDC Connector is now available in Confluent Hub. {{<tags/feature/ea>}}
+<!-- No docs needed-->
+
+* [Connection pooling](/stable/explore/going-beyond-sql/connection-mgr-ysql/). Added support for built-in connection pooling. {{<tags/feature/tp>}}
```

Comment:
this was TP in 2.19

Reply:
https://yugabyte.atlassian.net/jira/polaris/projects/IDEA/ideas/view/4493977?selectedIssue=IDEA-437&issueViewSection=comments - native connection pooling looks like a different idea which is TP now